### DISCUSSION TITLE
Welcome to OpenOrienteering Mapper discussions!

### DISCUSSION THREAD

**Discussion by dg0yt** [OpenOrienteering/mapper#1794] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by mlerjen** [comment]
For a while I am pondering on a way to cover the powerful draw and edit palette Mapper offers to the user.
I so far developed a symbol set to describe the various possible actions graphically.  Below a sample.


![image](https://user-images.githubusercontent.com/15611653/105466098-0e507080-5c94-11eb-96e1-6b8b6669e3fa.png)


I think in general the graphical language works.  What do you think? Would that be somehow helpful?

How could such be implemented here? - For instant I see three approaches.
a) Just integrate the graphical representations into the manual.
b) Make the overview to a poster (to print out) as an appendix to the manual 
c) Develop a third sample.omap to be integrated into the Mapper package. Beginners could just open the file and learn to draw and edit properly just while working with that file.

Your thoughts?

**Reply by ollesmaps** [reply]
That looks great. I would go with a) and c)
Could you share a picture with bigger resolution (zipped?).
Is there an alternative for each of these actions in Android?

**Reply by dg0yt** [reply]
Please re-post in a [new discussion](https://github.com/OpenOrienteering/mapper/discussions/new), probably category "Ideas". Thanks!

**Comment by Leonew0** [comment]
Hello! Who can help?) two days made a map, and now it writes an error and the map does not open (( mistake a program can't write a coordinates of map, one symbol

**Reply by dl3sdo** [reply]
If it has written the map to a file then it might be possible to fix the error by editing the map file.
You may send me your map directly to mapper AT dl3sdo.de

**Reply by Leonew0** [reply]
I'm sorry, I didn't see, how to attach a file to your site?

**Reply by dg0yt** [reply]
Attachments can be made to any comment. OMAP files aren't supported directly, but ZIP archives are fine. Please note that attachements are public - think twice if you really want to that.

**Comment by dg0yt** [comment]
Please stop adding comments to "Welcome to discussions": Open a new "discussion" or "issue" when you have a new question.
